products/amazon-rds-mysql.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Amazon RDS for MySQL
+category: db
+iconSlug: mysql
+permalink: /rds-mysql
+releasePolicyLink: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html
+
+releases:
+-   releaseCycle: "8.0"
+    releaseDate: 2018-10-23
+    eol: 2026-04-01
+    latest: "8.0.32"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.7"
+    releaseDate: 2016-02-22
+    eol: 2023-10-01
+    latest: "5.7.41"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.6"
+    releaseDate: 2013-07-01
+    eol: 2022-03-01
+    latest: "unknown"
+
+---
+
+> [Amazon RDS for MySQL](https://aws.amazon.com/rds/mysql) is a PaaS offering from Amazon for
+> creating managed MySQL Community Edition databases. RDS makes it easier to set up, operate, and
+> scale MySQL deployments on AWS cloud.
+
+**MySQL recommends that all users run the latest available minor release for whatever major version
+is in use.**
+
+- AWS will provide support for major releases 3 years after their RDS release date.
+
+- AWS will provide support for minor versions 1 year after their RDS release date.
+
+Keep in mind that by default minor versions are automatically upgraded during maintenance windows.
+
+For more info on how RDS versions are deprecated see the AWS [documentation](https://aws.amazon.com/rds/faqs/#What_happens_when_an_Amazon_RDS_DB_engine_version_is_deprecated.3F).
+
+Please follow [best practices](https://aws.amazon.com/blogs/database/best-practices-for-upgrading-amazon-rds-for-mysql-and-amazon-rds-for-mariadb) when performing upgrades to your RDS instance.
+
+### Minor Version Support
+
+| Release | Security Support | RDS Release |
+| ------- | ---------------- | ----------- |
```

Comment:
> I don't want to be making a PR every other month to update this minor table. So, I think I will omit this section since it's not even added as an API.

I think you are right.

[Your script](https://github.com/CodaBool/scrape-rds-eol/tree/main) can be integrated on https://github.com/endoflife-date/release-data, and it will generate a JSON file similar to https://github.com/endoflife-date/release-data/blob/main/releases/mysql.json. Unfortunately the format does not accept additional field yet, but that's something that should change : https://github.com/endoflife-date/release-data/issues/51 (we do not have any date to provide yet).

Reply:
The scripts I see there are all in Python. Would the script need to be in Python?

Reply:
I have now removed this Minor version table section in latest commit. Seems there is no current fit for this data.

Reply:
> The scripts I see there are all in Python. Would the script need to be in Python?

Nope, we also have Ruby scripts. I think @captn3m0 wanted to make things quite open regarding the language used, but let's ask him.

Reply:
Yes, I intentionally want to keep scripts simple, and standalone, and easy to write (read a few pages, and generate a few files).

Contributions are welcome in all major languages.